dev/io.openliberty.org.testcontainers/src/io/openliberty/org/testcontainers/generate/Dockerfile.java

```diff
@@ -122,34 +134,17 @@ private DockerImageName findBaseImageFrom(Path location) {
 
     }
     
-    private DockerImageName substituteBaseImage(final DockerImageName original) {
-        final String ARTIFACTORY_REGISTRY = System.getenv("ARTIFACTORY_REGISTRY");
-        
-        DockerImageName substituted = ImageNameSubstitutor.instance().apply(original);
-
-        if (original.equals(substituted)) {
-            System.out.println("Keep original BASE_IMAGE: " + original.asCanonicalNameString());
-            return original;
-        } else {
-            // Substitutor was used, also prepend the registry.
-            System.out.println("Use substituted BASE_IMAGE: " + substituted.asCanonicalNameString());
-            return substituted.withRegistry(ARTIFACTORY_REGISTRY);
-        }
```

Comment:
I see you moved
```
ImageNameSubstitutor.instance().apply
```
out of this method and into the constructor for setting `baseImageNameSubstituted`.

But the check for `if (original.equals(substituted))` is not done.  Is that no longer needed?

Reply:
I switched from using the Mirror substitutor to the full substitutor so all the logic for appending the registry is handled there.

Reply:
Thanks!